core/packages/test/.envrc-example

```diff
@@ -1,19 +1,23 @@
 ## Config for E2E stack
 
-export POLKADOT_BIN=/path/to/polkadot/target/release/polkadot
```

Comment:
I don't think we should remove this, it's useful to have the polkadot instance separate from the project.

Reply:
+1 for this comment. I find it useful to have `polkadot` repo outside of the project repo. It is easier to clean and rebuild the project this way. Although the changes below which fetch the latest polkadot on demand could be useful for test runners.